jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpContentWrapper.java

```diff
@@ -24,13 +25,18 @@
  */
 public class HttpContentWrapper implements HttpContent
```

Comment:
Can we move this class as an inner class of `HttpContent`? To be consistent with all other wrappers we have.

Reply:
done